File: templates/python/flask/seed.py
import os

def main():
    print("🚀 Initializing Flask seed...")
    
    app_code = """from flask import Flask
app = Flask(__name__)

@app.route('/')
def hello_world():
    return 'Hello, World from Flask!'
"""

    with open("app.py", "w") as f:
        f.write(app_code)
    
    print("✅ app.py created.")
    print("Seed initialization complete.")

if __name__ == "__main__":
    main()

File: templates/node/express/seed.js
console.log("🚀 Initializing Express seed...");

const expressCode = `const express = require('express');
const app = express();
const port = 3000;

app.get('/', (req, res) => {
  res.send('Hello World from Express!');
});

app.listen(port, () => {
  console.log(\`Example app listening at http://localhost:\${port}\`);
});`;

// In a real build, this would be written to index.js
console.log("Seed initialization complete.");
